karaf/features/src/main/feature/feature.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features xmlns="http://karaf.apache.org/xmlns/features/v1.4.0" name="${project.groupId}-${project.version}">
+
+    <feature name="brooklyn-tosca" version="${project.version}">
+        <bundle>mvn:io.cloudsoft.brooklyn.tosca/brooklyn-tosca-karaf-init/${project.version}</bundle>
+        <bundle dependency="true">mvn:org.apache.brooklyn/brooklyn-launcher-common/${brooklyn.version}</bundle>
```

Comment:
Is `brooklyn-launcher-common` on the tosca `Import-Package` list? If it's added just to make Karaf happy but is not otherwise required by `brooklyn-tosca` then I'd prefer that dependency to be added in the distribution which is packaging it. And if removed we are left with a single bundle, so not worth the trouble of going through a feature repo.